### ISSUE TITLE
Lack of confirmation messages

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1536]
### Description

There is currently no notification or confirmation message displayed when a user shares a document; this might lead to confusion to certain users

### Steps to reproduce

1. Open Share modal & share the document with an user/team
2. There isn't any type of confirmation that the event actually happened

### Expected behavior

There should be some kind of confirmation after sharing the document. This could be either a confirmation notification, an icon that confirms the event etc.

### Which operating system are you using?

Microsoft Windows

### What browsers are you seeing the problem on?

Mozilla Firefox

### Extensions

.

### Version

2024.6.0
